### ISSUE TITLE
Grouped devices blocks only have a block id 

### ISSUE THREAD

**Issue by pquinqu** [Dashticz/dashticz#1177]
Hi,
trying this type of construct in the CONFIG.js file:

`blocks['swimming_pool'] = { blocks: ['swimming_pool_mode','swimming_pool_temperature'] }`

will resolve as a div block like that:
```
<div id="block_31">
    <div id="block_32"><div data-id="swimming_pool_mode" ...</div>
    <div id="block_33"><div data-id="swimming_pool_temperature" ...</div>
</div>
```
My problem: i'm unable to identify the block 'swimming_pool' by a data-id="swimming_pool" in the custom.css file so i can't style the block by it's name.
Used in a popup, block numbers are dynamic so impossible to style a dynamic block id at all.

I don't know if this behavior is the wanted one, just need help.
Thank you much.

EDIT: unable to add a class with the AddClass modifier in the surrounding block instead.

**Comment by JRB202** [comment]
Use this kind of syntax in config.js and inside function afterGetDevices(){
CurrBlock = 'swimming_pool_mode';
$('.block_'+CurrBlock).addClass('border_blue');

**Comment by lokonli** [comment]
or:

Add the addClass parameter to the block definition

```
   blocks['myblock'] = {
      addClass: 'mynewclass',
   }
```

and define mynewclass in custom.css:

```
.mynewclass {
      background-color: 'yellow'!important
}
```

or use data-id as selector in custom.css:
```
[data-id='swimming_pool_mode'] .mh {
   background-color: 'yellow'!important
}
```
(it can be you have to remove the space before .mh)(I can't test at the moment)

**Comment by pquinqu** [comment]
Hi,
As i wrote earlier, the issue is not on the 'swimming_pool_mode' block or the 'swimming_pool_temperature' block.
The issue is : when a block (ie the 'swimming_pool' in the example) includes other blocks in a construction like block['blockname]: { blocks: ['name_x','name_y']}
the block['blockname'] as shown in the exemple, has no "blockname" data-id.
and trying addClass:'someclass', don't work in that case also (in fact, none of the blocks keywords are working) 

So it's impossible to style the block via custom.css in the case of using this type of construction in a popup (block ids are dynamic and data-id don't exist)

All your exemples are working fine, but it's not my use case.

**Comment by lokonli** [comment]
Solved with #1180 

See:
https://dashticz.readthedocs.io/en/beta/blocks/domoticzblocks.html#grouped-devices

**Comment by pquinqu** [comment]
I can confirm that PR #1180 commited on beta branch solves the issue. Closing.
